### PR TITLE
update replies

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/components/BlockchainList/CommentWithReplies.tsx [hunk 1]
```diff
@@ -71,12 +71,11 @@ function makeComment(post?: PostStructOutput, cast?: Cast) {
 
   if (cast) {
     return {
-      id: cast.merkleRoot,
-      replyToId: cast.merkleRoot,
-      replier: `@${cast.body.username}`,
-      replierAddress: cast.body.address,
-      content: cast.body.data.text,
-      timestamp: (cast.body.publishedAt / 1000) ^ 0,
+      id: cast.hash,
```
Comment (T-Damer) [inline src/components/BlockchainList/CommentWithReplies.tsx:74]: Let's put `cast.hash` into `const`

File: src/components/BlockchainList/CommentWithReplies.tsx [hunk 2]
```diff
@@ -71,12 +71,11 @@ function makeComment(post?: PostStructOutput, cast?: Cast) {
...
+      replyToId: cast.hash,
+      replier: `@${cast.author.username}`,
+      content: cast?.text,
+      timestamp: (cast.timestamp / 1000) ^ 0,
```
Comment (T-Damer) [inline src/components/BlockchainList/CommentWithReplies.tsx:78]: Shouldn't we use `timestamp` here?

Comment (T-Damer) [inline src/components/BlockchainList/CommentWithReplies.tsx:78]: We should extract those calculations into `helper`

Comment (MixailE) [reply src/components/BlockchainList/CommentWithReplies.tsx]: Here we need to refactor the application, it takes more time, let's add a task

**Review by upacyxou — APPROVED** [review]

**Pull request merged**